nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/Size.java

```diff
@@ -57,11 +57,6 @@ public String onnxName() {
         throw new NoOpNameFoundException("No onnx name found for shape " + opName());
     }
 
-    @Override
-    public String tensorflowName() {
-        return "Size";
-    }
```

Comment:
Why remove this? `Size` is a legitimate TF operation, and it's not (as far as I can see) mapped to any other java operation class for import.
https://github.com/eclipse/deeplearning4j/blob/master/nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/resources/ops.proto#L30364